src/components/Form/CrossPodWriteForm.jsx

```diff
@@ -97,51 +109,98 @@ const CrossPodWriteForm = () => {
             htmlForAndIdProp="upload-doc"
             handleDocType={handleDocType}
             docType={docType}
           />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-expiration">Expiration date (if applicable): </label>
-          <input id="upload-doc-expiration" name="date" type="date" />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doc-desc">Enter description: </label>
           <br />
+          <FormControl>
+            {/* TODO: Determine whether InputLabel below is necessary or redundant */}
+            <InputLabel htmlFor="upload-doc-expiration" />
+            <LocalizationProvider dateAdapter={AdapterDayjs}>
+              <DatePicker
+                id="upload-doc-expiration"
+                name="date"
+                format="MM/DD/YYYY"
+                label="Expire date (if applicable)"
+                value={expireDate}
+                onChange={(newExpireDate) => setExpireDate(newExpireDate)}
+                type="date"
+                slotProps={{
+                  textField: {
+                    helperText: 'MM/DD/YYYY'
+                  }
+                }}
+              />
+            </LocalizationProvider>
+          </FormControl>
           <br />
-          <textarea id="upload-doc-desc" name="description" {...description} />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="upload-doctype">File to upload: </label>
-          <input
-            id="upload-doctype"
-            type="file"
-            name="uploadDoctype"
-            accept=".pdf, .docx, .doc, .txt, .rtf, .gif"
-            onChange={handleFileChange}
-          />
-          <button disabled={state.processing} type="submit">
-            Upload file
-          </button>
-        </div>
-      </form>
+          <FormControl fullWidth>
+            {/* TODO: Determine whether Typography below is necessary or redundant */}
+            <Typography htmlFor="upload-doc-desc">Enter description:</Typography>
+            <TextField
+              id="upload-doc-desc"
+              name="description"
+              multiline
+              rows={4}
+              label="Enter Description"
+              value={docDescription}
+              onChange={(newDocDescription) => setDocDescription(newDocDescription.target.value)}
+              placeholder="help"
+            />
+          </FormControl>
+          <br />
+          <br />
+          <FormControl fullWidth>
+            <Button
+              variant="contained"
+              component="label"
+              color="primary"
+              id="upload-doctype"
+              name="uploadDoctype"
+              onChange={handleFileChange}
+              required
+            >
+              Choose file
+              <input type="file" hidden accept=".pdf, .docx, .doc, .txt, .rtf" multiple />
+            </Button>
+            <FormHelperText>
```

Comment:
In the case of long file name, we need a way to truncate it so that the form would not stretch. I believe styling this with sx should fix the problem.